packages/experiments/src/components/Pagination/examples/Pagination.Basic.Example.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { Pagination } from '../Pagination';
+
+export interface IPaginationBasicExampleState {
+  selectedPageIndex: number;
+}
+export class PaginationBasicExample extends React.Component<{}, IPaginationBasicExampleState> {
+  constructor(props: {}) {
+    super(props);
+
+    this.state = { selectedPageIndex: 0 };
+  }
+
+  public render(): JSX.Element {
```

Comment:
Would be great to have in the example an actual use of pagination with some sort of pages changing every time you select a different one.

Reply:
the selected page button does have different style and is there any other change you would like to see? 

---
In reply to: [221848933](https://github.com/OfficeDev/office-ui-fabric-react/pull/6532#discussion_r221848933) [](ancestors = 221848933)